data/sql_setup/prepare_schedule_a.sql

```diff
@@ -1,3 +1,15 @@
+-- Create table to hold sub_ids of records that fail during the nightly
+-- processing so that they can be tried at a later time.
+-- "sub_id" is not unique as we may have to process it multiple times.
+-- The "action" column denotes what should happen with the record:
+--    insert, update, or delete
+drop table if exists ofec_sched_a_nightly_retries;
+create table ofec_sched_a_nightly_retries (
+    sub_id numeric(19,0) not null primary key,
```

Comment:
D'oh!  When I had worked through this more I realized we shouldn't ever have a `sub_id` in this table more than once, so I figured the `primary key` would be okay.  I'll update the comment to reflect that, thanks!